docs/addon-linkerd

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+title: "Linkerd addon"
+permalink: /docs/addon-linkerd
+---
+
+# linkerd
+
+Enabling this addon will deploy the necessary services and proxies for the
+linkerd service mesh. To use linkerd effectively it is also required to set up
+and configure the `linkerd` CLI utility.
+
+```bash
+microk8s.enable linkerd
+```
+
+The [linkerd documentation][linkerd-docs] suggests the following for installing
+the command line tool:
+
+```bash
+curl -sL https://run.linkerd.io/install | sh
```

Comment:
There is a `microk8s.linkerd` command. You will not need to do this.

Reply:
cool, well, that's useful to know